instrumentation/netty/netty-4.1/src/main/java/io/opentelemetry/javaagent/instrumentation/netty/v4_1/AttributeKeys.java

```diff
@@ -8,51 +8,19 @@
 import io.netty.util.AttributeKey;
 import io.opentelemetry.api.trace.Span;
 import io.opentelemetry.context.Context;
-import io.opentelemetry.javaagent.instrumentation.api.WeakMap;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentMap;
 
 public class AttributeKeys {
-  private static final WeakMap<ClassLoader, ConcurrentMap<String, AttributeKey<?>>> map =
-      WeakMap.Implementation.DEFAULT.get();
-  private static final WeakMap.ValueSupplier<ClassLoader, ConcurrentMap<String, AttributeKey<?>>>
-      mapSupplier =
-          new WeakMap.ValueSupplier<ClassLoader, ConcurrentMap<String, AttributeKey<?>>>() {
-            @Override
-            public ConcurrentMap<String, AttributeKey<?>> get(ClassLoader ignore) {
-              return new ConcurrentHashMap<>();
-            }
-          };
 
   public static final AttributeKey<Context> CONNECT_CONTEXT_ATTRIBUTE_KEY =
-      attributeKey(AttributeKeys.class.getName() + "connect.context");
+      AttributeKey.valueOf(AttributeKeys.class, "connect.context");
 
   // this attribute key is also used by ratpack instrumentation
   public static final AttributeKey<Context> SERVER_ATTRIBUTE_KEY =
-      attributeKey(AttributeKeys.class.getName() + ".context");
+      AttributeKey.valueOf(AttributeKeys.class.getName() + ".context");
```

Comment:
```suggestion
      AttributeKey.valueOf(AttributeKeys.class, "server-span");
```

Reply:
oops, fixed, and I like the naming suggestions